tools/gdb/gdb_scripts/hazard_pointers.py

```diff
@@ -0,0 +1,80 @@
+# Helper scripts for hazard pointers that can be called in gdb.
+# Example: 
+#     (in gdb) 
+#     source hazard_pointers.py
+#     dump_hazard_pointers`
```

Comment:
Dangling `

Reply:
Updated in [85bdb20](https://github.com/wiredtiger/wiredtiger/pull/9801/commits/85bdb20849cbd8afde6506bf1d480f568de9340e)